src/components/Button/index.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 
 interface ButtonProps {
   name: string;
+  dataTestId?: string;
   disabled?: boolean;
   className?: string;
 }
```

Comment:
Should we add `...htmlAttributes` at the end of the Button function arguments instead?
Then when we need another HTML attribute, we don't need to add it here, except for the custom one.

```suggestion
interface ButtonProps {
  name: string;
  disabled?: boolean;
  className?: string;
}
```